### DISCUSSION TITLE
Don't release war effort.

### DISCUSSION THREAD

**Discussion by ghost** [chromiecraft/chromiecraft#3736] [General]
It's boring and pointless since this is Wrath of the Lich King and adds nothing to AzerothCore. Skip it and just open the instance instead. The faster we get to The Burning Crusade the better and more fun the game will be.

Regards.

PS: Allow World Buffs on Molten Core and Onyxia. These have been already tested and it allows pugs to be more successful at finishing the raids. It's a game mechanic as World Bosses are, so don't remove content just because you feel like it please. I don't need World Buffs to farm Murlocs Thanks.

**Comment by djobf** [comment]
Hard disagree, war effort and the Scepter questline are fun IMO. Why wouldn't it add nothing to AC ? It's content like everything else, which needs to be played, tested and fixed.
Ofc, the effort should be scaled to the size of the server though.
The raids are easy enough at is is without WBs, if PUGs don't even sometimes wipe on raids, then what's even the point...
Imo they removed WBs to avoid the raid logging meta and most people I've encountered seem ok with it

**Comment by RestoCC** [comment]
> It's boring and pointless since this is Wrath of the Lich King and adds nothing to AzerothCore. Skip it and just open the instance instead. The faster we get to The Burning Crusade the better and more fun the game will be.

>It's a game mechanic as World Bosses are, so don't remove content just because you feel like it please. I don't need World Buffs to farm Murlocs Thanks.

So you want content removed to suit you, but other content to not be removed to also suit you?

I hard agree with Djobf earlier, its an MMO still and the questline promotes a multitude of benefits.

1. Content
2. Gives Devs time to polish AQ20/40
3. Guilds reason to work together and actively play, particularly during a heavy raid-logging season of the server
4. Gold sink
